tests/test_command.py

```
@@ -407,6 +410,32 @@ def test_submit_script_location_local_java(self):
             with self.assertRaises(qds_sdk.exception.ParseError):
                 qds.main()
 
+    def test_submit_sql(self):
+        sys.argv = ['qds.py', 'sparkcmd', 'submit', '--sql', 'show dummy', '--language', 'sql']
+        print_command()
+        Connection._api_call = Mock(return_value={'id': 1234})
+        qds.main()
+        Connection._api_call.assert_called_with('POST', 'commands',
+                    {'macros': None,
+                     'label': None,
+                     'language': 'sql',
+                     'tags': None,
+                     'name': None,
+                     'sql': 'show dummy',
+                     'program': None,
+                     'cmdline':None,
+                     'command_type': 'SparkCommand',
+                     'arguments': None,
+                     'user_program_arguments': None,
+                     'can_notify': False,
+                     'script_location': None})
+
+    def test_submit_sql_no_program(self):
+        sys.argv = ['qds.py', 'sparkcmd', 'submit', '--sql', 'show dummy']
```

This should be the positive test and the other one should raise an error.